.github/workflows/orion.yaml

```
@@ -9,14 +9,83 @@ env:
   CI_ROOT: ${{ secrets.ORION_CI_ROOT }}  # Env. secret variable
 
 jobs:
-  validate-user:
+  authenticate:
     runs-on: [self-hosted, orion-ready]
-    timeout-minutes: 100
+    timeout-minutes: 10
 
     steps:
     - name: Checkout ci-workflows
       uses: actions/checkout@v3
+      with:
+        repository: TerrenceMcGuinness-NOAA/ci-workflows
+
+    - name: Validate global-workflow user
+      run: |
+        scripts/validate_user.py -g ${{ github.actor }} -p ci-workflows -d ${{ env.CI_ROOT }}
```

```suggestion
        scripts/validate_user.py -g ${{ github.actor }} -p global-workflow -d ${{ env.CI_ROOT }}
```
This should be validating if the `github.actor` is authorized to execute the CI in project `global-workflow`

---

True, but the path itself is coming from the CI-workflow repo (`_worker/ci-workfow/ci-workflow`), but from a nomenclature point of view, we can change it since we are pointing to the file by hand.